ayup/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,19 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final ImmutableList<String> FUN_FACTS = ImmutableList.of(
```

Comment:
I guess if you didn't want to bring in the Guava dependency you'd do something like...

```java
private static final ArrayList<String> FUN_FACTS =
      new ArrayList<>(Arrays.asList("fact 1", "fact 2", ...));
```

Guava is used in basically every Java project at Google.